win32ss/user/user32/controls/listbox.c

```diff
@@ -322,8 +322,12 @@ static LRESULT LISTBOX_SetTopItem( LB_DESCR *descr, INT index, BOOL scroll )
         else
             diff = (descr->top_item - index) * descr->item_height;
 
-        ScrollWindowEx( descr->self, 0, diff, NULL, NULL, 0, NULL,
-                        SW_INVALIDATE | SW_ERASE | SW_SCROLLCHILDREN );
+        if (descr->style & LBS_MULTICOLUMN)
+            ScrollWindowEx(descr->self, diff, 0, NULL, NULL, 0, NULL,
+                           SW_INVALIDATE | SW_ERASE | SW_SCROLLCHILDREN);
+        else
+            ScrollWindowEx(descr->self, 0, diff, NULL, NULL, 0, NULL,
+                           SW_INVALIDATE | SW_ERASE | SW_SCROLLCHILDREN);
```

Comment:
Since this is wine-synced can you put this inside an #ifdef similar to this?
`#ifdef  __REACTOS__`
`// New Code here`
`#else`
`// Old Code here`
`#endif`